### PR TITLE
Adding Black Jack Environment

### PR DESCRIPTION
**Pull request by RajGhugare19**

### CODE REVIEW

**Comment by RajGhugare19** [comment]
I have just added the blackjack for now to review if I am doing it the correct way. Others will follow shortly :)

File: src/environments/toytext/blackjack.jl [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+This environment corresponds to the version of the blackjack problem
+described in Example 5.1 in Reinforcement Learning: An Introduction
+by Sutton and Barto.
+http://incompleteideas.net/book/the-book-2nd.html
+"""
+
+function BlackjackEnv(;natural=false,rng = Random.GLOBAL_RNG)
+    # natural = flag to payout on a "natural" blackjack win, like casion rules
+    action_space = DiscreteSpace(2)
+    observation_space = VectSpace((DiscreteSpace(32),DiscreteSpace(11),DiscreteSpace(2)))
```
Comment (findmyway) [inline src/environments/toytext/blackjack.jl]: Could you use MultiDiscreteSpace here?
The type will be more concise.

**Review by findmyway — COMMENTED** [review]
Could you also add a test case to make sure that this environment can pass some basic tests?

File: src/environments/toytext/blackjack.jl (hunk 1 above)
Comment (RajGhugare19) [reply src/environments/toytext/blackjack.jl]: Alright I'll change this to MultiDiscreteSpace

**Comment by RajGhugare19** [comment]
> Could you also add a test case to make sure that this environment can pass some basic tests?
Could you give some example? and could you tell me in which file shall I add these tests?

**Comment by findmyway** [comment]
See examples here:

https://github.com/JuliaReinforcementLearning/ReinforcementLearningEnvironments.jl/blob/master/test/environments.jl#L32

**Comment by RajGhugare19** [comment]
Is this right? @findmyway

**Pull request merged**